webapp/src/css/enketo/medic.less

```diff
@@ -472,8 +478,8 @@
         font-size: @font-medium;
         justify-content: center;
         display: flex;
-        .error_msg {
-          color: #a94442;
+        .loading-status {
```

Comment:
Matches other places in the app where the loading status is greyed out (instead of being red).